### DISCUSSION TITLE
Flow Cell Plumbing Quesitons & Other Odds & Ends

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#904] [Q&A]
While awaiting an ORPprobe and a pH probe to arrive, I am trying to determine a reasonable location for the flow cell taps.  I  will be using 90 degree John Guest elbow taps as suggested in the Wiki.  It looks like my pool installer couldn't run 2 feet of PVC without having to add a connector.  Of course there are no unions to be found anywhere other than the salt cell, but that's a challenge for a different day.  Such fun!

Anyway, below is a picture of the plumbing from the filter to/from the gas heater & on to the salt cell.  

For the supply line to the flow cell, it looks like I can either install the tap in the PVC from the filter directly or through the collar in the connector by the arrow.  I understand, in general, one should place the tap on a thicker section of PVC to reduce the likelihood of cracking the line. For the flow cell supply line, that's going to be a challenge.  Do you have any suggestions as to its placement?

For the return line from the flow cell, there are a few options: the 'T' fitting for the heater return line, the 90 degree bend connector to the salt cell, through a connector collar, or in the unblemished PVC above the salt cell.  Any thoughts for placement are appreciated!

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/11eb4f1f-9b19-4c7c-9c8d-dec7f66632bb)

My other questions relate to the acid feeder.  It's a Model 45M3 Stenner pump and is capable of delivering 2.92 - 57.85 ml/minute in 2-1/2% increments.  I have not yet  calibrated its actual output.  The pump is currently at the lowest setting (2.92 ml/minute) and the pump runs the entire time the filter pump is on and the salt cell is running.  Typically, it's 12 hours/day.

Assuming the flow rate boundaries and adjustable increments are accurate what setting should I use initially for the pump?  The pool is  20,000 gallons and I plan to use/dilute the acid to 14.5 Baume and dose the acid by volute as suggested in the Wiki.

Thanks for your help!

**Comment by tagyoureit** [comment]
For the flow cell, you want it as close to the pool as possible so you are measuring the values coming from the pool - not what's affected by your filter or acid pump, etc.  I eventually hooked my flow cell up to an output on the pump itself, which is very convenient.  You may have an extra port there.

And for the Stenner, you input the ml/min into REM so we will adjust the timing accordingly.  Personally, I can hear my pump so I'd turn it up quite a bit so it isn't running all day.  If you find there are too many swings in the pH you can turn the rate down a bit but I don't think you'll have that problem.

**Reply by FlaMike** [reply]
The pump is a Pentair IntelliflowXF VSF.  It has two ports.  One is under the under the strainer pot that I think is a drain plug.  The other may be on the supply side.  Picture below.  The strainer pot is on the right.

Can I use one to  supply the flow cell & the other for its return?    If you happen to know the size of the fittings, I would be happy to get  John Guest plugs for them & not have to drill taps in the PVC.  The pump output line does have elbows and 45 degree couplings I could use for tap locations for the supply.

![PumpFittings](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/347e0345-4cdd-469a-9f5d-667a628b9b71)


Can you  recommend a strainer for the flow cell supply line if I tap the supply line prior to the filter?

Thank you so much your your help!!!!  I'm gonna get  there soon, I hope :-)

**Reply by tagyoureit** [reply]
This size is 1/4" NPT.  
I can't remember what filter I used but you can try something like [this](https://www.amazon.com/Line-Strainer-stainless-filter-screen).

You _may_ be able to use both ports.  I'm not sure the pressure differential between them.  I would also use a [check valve](https://www.johnguest.com/us/en/od-valves/acetal/check-valve) to keep the water going in one direction.

I also used a [shut off valve](https://www.johnguest.com/us/en/od-valves/polypropylene/straight-shut-off-valve-speedfit-to-speedfit) before the flow cell which is handy for maintenance but also you can use it as a pressure regulator because the ports will be pushing more water through then you need/want.

**Reply by FlaMike** [reply]
Thank you!  I will order the parts as you suggest & see if the pressure differential is sufficient to keep the flow cell & flow switch happy.  I'll post whether or not it works so others may benefit from my experience.  It will likely be at least a week until everything gets here & I have time to play with it.

**Reply by FlaMike** [reply]
@tagyoureit Quick question for you...  When you switched to using a port on the pump for input to your flow cell, did you keep return line from the flow cell to the pool return line after the heater as discussed in the Wiki?  I will try using both ports on the pump, but want to have a backup plan in case the pressure differential is not great enough to meet the flow cell's needs.

Parts are all on their way.  Hoping to play with them this weekend.

Thanks!

**Reply by tagyoureit** [reply]
Mine was after the salt cell, but not the heater.  But it's such a small amount of water you can add it back to the main lines wherever is convenient.

**Comment by FlaMike** [comment]
Thank you for the info.  I'm still planning to try using the 2 ports on the
pump but there are 90 degree elbows before & after the salt cell if the
pressure differential isn't sufficient.  Good to have a backup plan!

**Reply by Exit2Studios** [reply]
@FlaMike please keep us posted. I’m very curious if this will work. I currently use one of those ports to fill my pump basket for priming.

**Reply by FlaMike** [reply]
Will do.  Still waiting on some of the plumbing parts suggested above to show up.  I also need to attach the flow cell to the wall & need a friend's impact drill to do it--house is all concrete & masonry.  Hope to run this experiment within the next few days.

**Reply by FlaMike** [reply]
It works!!!!!   Using the pump's two drain ports can feed and drain the flow cell successfully!!!

I have not had an opportunity to calibrate & install either the pH or ORP probes, but I installed the flow cell, and plumbed it as shown below.  The inflow line is black and connected to the pump's "output" drain port (on the left in picture a few posts up).  The outflow line is blue and has a check valve to prevent backflow into the cell.  The outlfow line connects to the pump's basket strainer drain port (on the right in the same picture).  

Per the suggestions from @tagyoureit I installed a shutoff valve inline on the inflow line so I can clean/change the filter just above it in the picture.

It's actually pretty easy to throttle the flow into/out from the cell using the inline shutoff (although the flow cell's input valve would likely work just as well) and the cell's outflow valve.  Once the probes are installed, I assume I can just use one of the input valves to control the flow rate & leave the outflow valve opened fully.  (Without the probes in place opening the outflow valve fully drains the cell pretty quickly.)  Anyway, you can easily see the flow switch float moving up so the magnet closes the (presumed) reed switch.  The proof is in the pudding on the screenshot from REM.

In short, if you have an Intelliflow VSF pump with two drain ports, you can use them to feed & drain the flow cell without any need to drill & tap PVC pipes!!!!!  A huge win as far as I'm concerned.

I plan to write up my ET to Nixie conversion as well as my DIY REM CHEM once the project is complete.  I will do so to complement the excellent posts from @CodeSlinger69 and the Wikis, not to usurp them.  Without that documentation and the fabulous support from this group, I would not have been able to attempt, let alone succeed, with my build.

BTW, the picture of the flow cell is crooked, not the flow cell itself (I checked with a level before securing it fully).

One question/concern ahead...  It appears a fair portion of the storage solutions in the probes leaked out before I received them from Atlas Scientific.  There is not enough solution to keep either of the probe tips wet regardless of orientation.  Are these probes compromised?  I've not done anything other than open the packages to examine the cells.

Once again, thanks to all who have guided me throughout my journey!!



![PXL_20240304_213100446](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/10301535-9a8c-441c-b5e1-be198d7dab69)


![PXL_20240304_213110033](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/a4d8bc02-a6f5-4c2d-aca7-09a95ca713f9)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/cca1d65d-4e53-44b0-aa0f-236ea4b8f254)

**Reply by Exit2Studios** [reply]
That’s awesome. Would love to see a BOM for what you used.

**Reply by tagyoureit** [reply]
Looks great!  Yes, the extra shut off valve would be redundant. I forgot the flow cell already included them.

For the probes, in theory, they should always stay wet. If you think it leaked out in transit and it's only been a few days, it probably isn't an issue. If you think they've been dry for sometime or just want some peace of mind, I would double check with Atlas.

**Reply by FlaMike** [reply]
@Exit2Studios here is the BOM for my flow cell setup.  Hope it helps!  Be sure to specify 3/8" OD fittings for everything.

IPS Flow Cell:  https://www.poolblu.com/flow-cell-with-flow-switch
John Guest Shut Off Valve: https://www.amazon.com/dp/B003YKF2F6?psc=1&ref=ppx_yo2ov_dt_b_product_details
John Guest Check Valve:  https://www.amazon.com/dp/B005XTY90A?psc=1&ref=ppx_yo2ov_dt_b_product_details
John Guest 3/8" OD Tubing:  https://www.amazon.com/dp/B06XG1SMTN?psc=1&ref=ppx_yo2ov_dt_b_product_details
Tailonz 1/4" NPT Elbow (Need 2 for pump drains): https://www.amazon.com/dp/B0885ZKHR2?ref=ppx_yo2ov_dt_b_product_details
Teflon Tape: https://www.amazon.com/dp/B0CBRSQLDN?psc=1&ref=ppx_yo2ov_dt_b_product_details
Inline Filter: https://www.freshwatersystems.com/products/neofit-acetal-gray-filter-strainer-3-8-tube-200-mesh?syclid=3ba1f876-01d4-480b-8337-ba660c492379&variant=41627412922552
Extra Screen for Inline Filter: https://www.freshwatersystems.com/products/neofit-200-mesh-replacement-screen-for-afs-hbmu-series?variant=42995517751480

Be certain to double-check direction of arrows on the filter & check valve to be sure the water flow is in the correct direction!!  (For the record, I did *not* mess it up, but I'm sure it could prove to be frustrating if I had!)

**Comment by FlaMike** [comment]
I'll be happy to post it tomorrow.  Very little magic, actually.  Mostly
what was suggested by Russ above.  Worked out better than I expected!

**Comment by FlaMike** [comment]
Thanks.  Your suggestions made it possible and I truly appreciate them.
The extra shutoff isn't a problem & may prove handy to kill the flow when
cleaning the filter screen without turning off the pump.  It was only about
$7.00, so what the heck.

I'll contact Atlas tomorrow about the probes.  It may be several more days
before I install them due to weather & everyday life.

**Reply by tagyoureit** [reply]
Keep us posted if the extra filter is needed.  Curious to see how much it might catch.

**Reply by FlaMike** [reply]
I can report the filter did its job.  Sadly there still appears to be sufficient concrete dust in the pool from some deck work we're having done that it clogged the little filter after only a couple of days to the point the float switch contact opened.  I removed the filter & ewe!  A bunch of very fine particulate had been trapped.  Easy enough to rinse out & all is good again.

I did notice a small amount of rust on one rib of the filter.  I'm surprised & a bit concerned because it's encased in plastic.  Hoping its superficial & not a long term problem with the filter's construction or my pool water chemistry.  As I reported in this post, the CSI is not happy at the moment.  https://github.com/tagyoureit/nodejs-poolController/discussions/915

The filter I have is very similar to the one @Telemachus19777 used in this discussion.  https://github.com/tagyoureit/nodejs-poolController/discussions/514  Have you had any corrosion issues with your filter?

**Comment by FlaMike** [comment]
Will do!  It's this one...  200 mesh (60 micron) screen so there shouldn't
be anything substantial that gets past it.  I bought a spare screen.  The
shipping was as much as the parts so I invested in a backup rather than
getting hit a second time.

https://www.freshwatersystems.com/products/neofit-acetal-gray-filter-strainer-3-8-tube-200-mesh?syclid=3ba1f876-01d4-480b-8337-ba660c492379&variant=41627412922552

**Comment by FlaMike** [comment]
@tagyoureit I contacted Atlas about the lack of storage fluid in the caps & they responded as below very quickly.  Time to make up some nasty salt water & top off the caps until the rain ends & I can get the probes calibrated & installed.

Hi Michael,
 
Sorry to hear that.
Don't worry, the fix is easy.
 
Mix tap water and table salt until you believe the water is undrinkable.
Put the water in the storage cap.
That's all.
 
Our new probes take a while to be damaged from being dry (several weeks).

**Reply by FlaMike** [reply]
@tagyoureit   As posted above, I am using both ports on the Intelliflow to feed & drain the flow cell.  I installed an inline filter as you suggested to keep crud from reaching the flow cell since the water source is directly from the pump & not after the sand filter.

Every 2-3 days the inline filter I am using between the filter pump and the flow cell has clogged to the point that the flow switch trips.  Of course, if I don't see the no flow alarm for a while, the pH and/or ORP can get out of wack.  While I'm glad the filter is doing its job, I'm not happy about the daily maintenance I need to do to keep things happily chugging along.  I'm using a 200 mesh filter screen.  

Is there  a different filter you recommend or should I switch to a 100 or 40 mesh screen?  I'm also thinking about moving the input line from the filter pump to after the filter & before the heater, per the Wiki.  Still a little gun shy about drilling & tapping the PVC, but the current arrangement isn't a super solution.  If I move the input to the PVC line, I can keep the existing filter in place in case there's more crud that gets past the sand filter, but I'm not sure it's needed in that case.

Any recommendations including drilling & tapping techniques on PVC are welcome.   Thank you!

**Reply by tagyoureit** [reply]
So, for comparison and some food for thought...

- DE filters will filter down to 3-5 microns
- Cartridge filters will filter down to 10-20 microns
- Sand filters will filter down to 20-30 microns
(different sites estimates different trapping capabilities, of course)

A 200 mesh filter screen should filter down to 74 microns.  (100 mesh=149 microns or 40 mesh=400 microns)

Based off of this, (and not knowing your filter setup), you may still have a lot of construction debris floating around in the pool to pass through the filter (presumably) and then get trapped in the much more coarse mesh filter.  Or maybe you have a tear in the cartridges somewhere?  You might want to clean your filters a couple more times and see if that helps.  Or there is just a lot of debris that gets blown into your pool?

In any regard, I never had a filter screen at all and I ran the 1/4" tubing straight from the pump.  The probes can probably handle a good amount of abrasion, and if you keep the flow low enough it won't really damage the probes.  I wouldn't hesitate to remove the filter if you feel your pool is pretty clean and keep playing with the valves to a minimal amount of flow through the cell.

**Reply by FlaMike** [reply]
Thank you for your quick & helpful reply!  As always, your depth and breadth of knowledge are amazing.  I had no idea of the differences in particle size trapping for the different pool filters.  Our pool has an old school sand filter, so its limit is in the 20-30 micron range.  The 200 mesh inline filter is not as capable, but wow!  It has trapped a bunch of yucky stuff.

I agree we may have some grout and concrete dust yet to be removed from the pool.  I've not brushed the bottom for a few days, but I wouldn't be surprised to see underwater "dust"  clouds appear on the bottom when I do so shortly.  The filter pressure increases several PSI when I brush the pool and I continue to backwash it regularly as a result.  I also see the filter pump's GPM drop after brushing--another crud indicator.  I run extra Polaris cycles during and after brushing to get what it can as well as keeping things stirred up due the the Polaris' movement and the pump speed increase while the Polaris is running.

I agree the prudent approach is to continue brushing, backwashing, & cleaning the inline filter to the flow cell for a bit longer until things calm down.  At that point, I may remove the small inline filter & see how things go.

I also officially "opened" the pool a couple of days ago and have been swimming laps 1+ hours/day as a result.  The resulting agitation of the water seems to help stir up any debris in the pool, so that activity should help as well (in addition to reducing my winter girth!).  My wife has yet to dive in, so to speak, but I'm hoping to convince her yet again that 83 degrees is the perfect temperature for lap swimming :-)  For the moment, the pool is mine and mine alone.  I guess that's a good thing...
